src/Solocraft.cpp

```diff
@@ -42,10 +42,11 @@ class SolocraftConfig : public WorldScript
 
     void OnBeforeConfigLoad(bool reload) override
```

Comment:
```suggestion
    void OnBeforeConfigLoad(bool /*reload*/) override
```

Reply:
```C++
void OnBeforeConfigLoad(bool reload) override
{
    if (!reload)
    {
        // Load Configuration Settings
        SetInitialWorldSettings();
    }
    // Squelch compile warning
    (void) reload;

    // Load Configuration Settings
    SetInitialWorldSettings();
}
```
Sorry for the question, but if you comment out the parameter, then you can use it in the following lines? Anyway, I don't understand why you still want to make this change. If I had more information, it would be really helpful.